index.html

```diff
@@ -1,78 +1,84 @@
-}
- 
-#seite {
-    position: relative;
-    min-height: 100%;
-}
- 
-#inhalt {
-        padding-bottom: 2em;
-}
- 
-#fussbereich {
-    position:absolute;
-    bottom: 0;
-    width: 100%;
-    height: 2em;
-}
-		    
-            </style>
-    </div>
-		<div class="ui container" ng-view></div>
-	</body>
+
+<head>
+    <!-- MetaData: -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="language" content="english, en">
+    <meta name="keywords" content="ai,artificial intelligence,app,apk,open source,download,github,api,susi,susi ai,FOSSASIA">
+    <!-- Title: -->
+    <title>Meet Susi - your own personal assistant!</title>
+    <!-- Favicon:
+    <link rel="shortcut icon" href="./assets/images/.ico"> -->
+    <!-- Google-Fonts: -->
+    <link href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:100,300,400,600,700,900,400italic' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Cuprum:400,700' rel='stylesheet' type='text/css'>
+    <!-- Bootstrap core CSS -->
+    <link href="./assets/css/bootstrap.min.css" rel="stylesheet">
+    <link href="./assets/css/bootstrap-reset.css" rel="stylesheet">
+    <!--Animation css-->
+    <link href="./assets/css/animate.css" rel="stylesheet">
+    <!--Icon-fonts css-->
+    <link href="./assets/font-awesome/css/font-awesome.css" rel="stylesheet" />
+    <link href="./assets/ionicon/css/ionicons.min.css" rel="stylesheet" />
+    <!--Old Css and JS files-->
+    <!-- CSS Imports:
+    <link rel="stylesheet" href="./css/semantic.min.css">
+    <link rel="stylesheet" href="../css/custom.css"> -->
+    <!-- JavaScripts Import:
+    <script async defer src="https://buttons.github.io/buttons.js"></script>
+    <script async defer src="https://use.fontawesome.com/579dcd2e61.js"></script>-->
+    <!-- materialDeign Import for footer:
+    <link rel="stylesheet" href="https://storage.googleapis.com/code.getmdl.io/1.0.6/material.indigo-pink.min.css">-->
+    <!--Angular Files:-->
+    <script src="./assets/angular/js/angular.min.js"></script>
+    <script src="./assets/angular/js/angular-route.min.js"></script>
+    <script src="./assets/angular/js/main.js"></script>
+    <!-- Google Analytics Start -->
+    <!-- Google Analytics Stop -->
+    <!-- Custom styles for this template -->
+    <link href="./assets/css/style-frontend1.css" rel="stylesheet">
+    <link href="./assets/css/pace-loader.css" rel="stylesheet">
+</head>
+
+<body ng-controller="mainController">
+    <div ng-view></div>
+    <ng-include src="'./includes/footer.html'"></ng-include>
+    <!-- End Footer -->
+    <!-- js placed at the end of the document so the pages load faster -->
+    <script src="./assets/js/jquery.min.js"></script>
+    <script src="./assets/js/bootstrap.min.js"></script>
+    <script src="./assets/js/pace.min.js"></script>
+    <script src="./assets/js/wow.min.js"></script>
+    <!-- Jquery easing , SmoothScroll
```

Comment:
So does that mean we dont need these files that you added, like `assets/js/jquery-ui-1.10.1.custom.min.js` and `assets/js/jquery.easing.1.3.min.js` ?